.github/workflows/release-freeze.yml

```
@@ -9,14 +9,20 @@ on:
         options:
           - major
           - minor
+      freeze-commit:
+        type: string
+        description: Commit SHA to use for cut-off
+        required: false
+        default: main
 
 jobs:
   code-freeze:
-    uses: NVIDIA/NeMo-FW-CI-templates/.github/workflows/_code_freeze.yml@v0.12.0
+    uses: NVIDIA/NeMo-FW-CI-templates/.github/workflows/_code_freeze.yml@v0.17.3
     with:
-      library-name: NeMo Curator
+      library-name: NeMo-Curator
```

Would it not still be
```suggestion
      library-name: NeMo Curator
```
?